services/frontend-service/src/ui/components/button/button.tsx

```diff
@@ -37,11 +37,10 @@ export const Button = (props: { className?: string; label?: string; icon?: strin
             ref={control}
             aria-label={label || ''}>
             <div className="mdc-button__ripple" />
-            {!!icon && (
-                <i className="medium material-icons mdc-list-item__graphic" aria-hidden="true">
-                    {icon}
-                </i>
-            )}
+            {icon &&
```

Comment:
Why this change here ?

Reply:
It's as Tamer said. How it was before as doubling the complexity. Since the icon only needs the JSX Element, we don't need a div for it.